internal/node/fishtail/subscriber.go

```diff
@@ -0,0 +1,107 @@
+package fishtail
+
+import (
+	"encoding/json"
+
+	"github.com/eagraf/habitat-new/core/state/node"
+	"github.com/eagraf/habitat-new/internal/node/config"
+	"github.com/eagraf/habitat-new/internal/node/controller"
+	"github.com/eagraf/habitat-new/internal/node/hdb"
+)
+
+func NewFishtailIngestSubscriber(
```

Comment:
I want to be a bit careful with naming ... is this intended to be used by the fishtail service or by individual apps using fishtail? if the latter, then I don't think we should be pasing in `nodeConfig` and `nodeController` here. If the former, then isn't it subscribing to the firehose?